.circleci/config.yml

```diff
@@ -1276,7 +1277,8 @@ jobs:
             set -e
             export IN_CIRCLECI=1
 
-            cp -r /Users/distiller/pytorch-ci-env/workspace/. /Users/distiller/project
+            # copy with -a to preserve relative structure (e.g., symlinks), and be recursive
```

Comment:
otherwise git will say that `.dockerignore` (softlink to `.gitignore`) changed type